src/main/java/core/basesyntax/commands/Help.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.commands;
+
+class Help extends Command {
+
+    static final Help INSTANCE = new Help(
+            "help",
+            "[command]",
+            "Выводит в консоль "
+                    + "все доступные комманды и информацию к ним.");
+
```

Comment:
It would be better to create enum just for these Strings(including other commands) and not create static instance inside it's class